tests/test_models.py

```diff
@@ -234,3 +234,36 @@ def test_preprocessing(fmodel_and_data: ModelAndData) -> None:
         fmodel = fbn.models.base.ModelWithPreprocessing(
             fmodel._model, fmodel.bounds, fmodel.dummy, preprocessing
         )
+
+
+def test_transform_bounds_wrapper_data_format() -> None:
+    class Model(fbn.models.Model):
+        data_format = "channels_first"
+
+        @property
+        def bounds(self) -> fbn.types.Bounds:
+            return fbn.types.Bounds(0, 1)
+
+        def __call__(self, inputs: fbn.models.base.T) -> fbn.models.base.T:
+            return inputs
+
+    model = Model()
+    wrapped_model = fbn.models.TransformBoundsWrapper(model, (0, 1))
+    assert fbn.attacks.base.get_channel_axis(
```

Comment:
Could we here add `hasattr` checks for `data_format` and `not_data_format` like below. This assertion would also be true if `get_channel_axis` would just return `None` in both cases.

Reply:
Sure.